docs/_rpc/ns-net.md

```diff
@@ -0,0 +1,36 @@
+---
+title: net Namespace
+sort_key: C
+---
+
+The `net` API provides insight about the networking aspect of the client.
+
+* TOC
+{:toc}
+
+### net_listening
+
+Returns an indication if the node is listening for network connections.
+
+| Client  | Method invocation             |
+|:--------|-------------------------------|
+| Console | `net.listening`               |
+| RPC     | `{"method": "net_listening"}` |
+
+### net_peerCount
+
+Returns the number of connected peers.
+
+| Client  | Method invocation             |
+|:--------|-------------------------------|
+| Console | `net.peerCount`               |
+| RPC     | `{"method": "net_peerCount"}` |
+
+### net_version
+
+Returns the current ethereum protocol version.
```

Comment:
No, this returns the network id of devp2p, not the eth version. The one that returns the eth version was called `eth_version` and I removed it a long time ago because it was useless + ambiguous with Geth supporting 5 different versions at one point.